client/jetpack-connect/controller.js

```diff
@@ -205,6 +206,39 @@ export function authorizeForm( context, next ) {
 	next();
 }
 
+export function authorizeForm( context, next ) {
+	analytics.pageView.record( 'jetpack/connect/authorize', 'Jetpack Authorize' );
+
+	removeSidebar( context );
+
+	const { query } = context;
+	const validQueryObject = validator( authorizeQueryDataSchema )( query );
```

Comment:
This is now duplicated. We could extract to a helper function but I prefer to leave that for a follow-up PR which could focus on that. This PR is already quite extensive.